lib/rewards.js

```diff
@@ -23,7 +26,22 @@ export const runUpdateRewardsLoop = async ({ contracts, ethAddress, onMetrics })
   }
 }
 
-async function getScheduledRewardsWithFallback (contract, ethAddress) {
+async function getSparkRewardsScheduledRewardsWithFallback (ethAddress) {
+  try {
+    const res = await fetch(
+      `https://spark-rewards.fly.dev/scheduled-rewards/${ethAddress}`
+    )
+    const json = await res.json()
+    return typeof json === 'string'
+      ? BigInt(json)
+      : 0n // json can be `null`
```

Comment:
I was thinking about changing the service to return `"0"` also when the participant isn't known. This matches the behavior of the smart contract. Wdyt @bajtos?

Reply:
I don't have a strong opinion.

On one hand, as a REST API service client, I would expect the service to return 404 for entities (or URL paths) that don't exist.

On the other hand, a path with an address we haven't seen before is a valid entity, plus it makes the implementation on the client side easier, so it makes sense to return `0` in such case.